docs/modules/ROOT/pages/subscriptions/events/connect.adoc

```diff
@@ -0,0 +1,690 @@
+
+This means that, if the type targeted for the subscriptions defines two or more relationships in the schema and two or more relationships are created, there will be an equal number of events triggered as the number of created relationships. 
+
+Each event will have the relationships object populated with the created relationship's properties for one single relationship name only - all other relationship names will have a null value.
+
+The event will also contain the properties of the nodes at both ends of the relationship, as well as the properties of the new relationship, if any.
+
+NOTE: The `CONNECT` events represent new relationships being created and contain information about the nodes at each end of the new relationship. However, the connected nodes may or may not have previously existed. To subscribe to the node's updates, you need to use one of the `CREATE` or `UPDATE` subscriptions.
+
+== `CONNECT` event
+
+A subscription to a type can be made with the top-level subscription `[type]RelationshipCreated`. The subscription will contain the following fields:
+
+* `event`: The event triggering this subscription, in this case it will always be `"CONNECT"`.
+* `timestamp`: The timestamp in which the mutation was made. Note that multiple events may come with the same timestamp if triggered by the same query.
+* `<typename>`: The properties of the node target to the subscription. Only top-level properties, without relationships, are available. Note these are the properties before the operation that trigger the connection took place.
+* `relationshipFieldName`: The field name of the newly created relationship, as part of the node target to the subscription.
+* `createdRelationship`: An object having as keys all field names of the node target to the subscription which represent its relationships. For any given event, all field names but the one corresponding to `relationshipFieldName` will be null. The field name equal to `relationshipFieldName` will contain the relationship properties, if defined, as well as a `node` key containing the properties of the node on the other side of the relationship. Only top-level properties, without relationships, are available. Note these are the properties before the operation that trigger the connection took place.
+
+NOTE: Irrespective of the relationship direction in the database, the `CONNECT` event is bound to the type targeted for the subscription. The consequence is that - given a relationship between types A and B that is not reciprocal (that is, in the GraphQL schema type A defines a relationship to B but B does **not** define a relationship to A) and a GraphQL operation that creates the relationship between them - even though the two nodes will be connected in the database, only a subscription to type A will catch the `CONNECT` event. Check out the xref:subscriptions/events/connect.adoc#non-reciprocal-relationships[Non-reciprocal Relationships] section below for more details.
```

Comment:
I also note that the word reciprocal is used which is great.

Reply:
I preferred `reciprocal` over `bidirectional` as the latter might be associated with the idea of 1 relationship with arrows in both directions, and in this case it would be 1 relationship with one arrow in a single direction, irrespective of the type on either side.
For example
```graphql
type Movie {
  actors: [Actor!]! @relationship(direction: "IN", type: "ACTED_IN")
}
type Actor {
  movies: [Movie!]! @relationship(direction: "OUT", type: "ACTED_IN")
}
```
There is only 1 type of relationship here: (a: Actor)-[:ACTED_IN]->(m: Movie)
So both `actors` and `movies` fields are representative of the same relationship with the same direction, just on the different types.
I'm worried that if we were to use the term `bidirectional` there might be a confusion with something like:
(a: Actor)<-[:ACTED_IN]->(m: Movie), even though this is not valid in Neo4j.

Reply:
Oh, that's a great point! I agree. Let's skip over this suggestion :)